themes/vue/source/css/page.styl

```diff
@@ -14,6 +14,7 @@
 @import "_style-guide"
 @import "_modal"
 @import "_scrimba"
+@import "_vuemastery-banner"
 @import "_vue-mastery"
```

Comment:
For https://github.com/vuejs/vuejs.org/issues/2515 I suspect this line needs to match the file rename with `@import _vuemastery` or revert the filename change. Explains an empty https://vuejs.org/css/page.css anyways. Thanks!